DependencyInjection/Configuration.php

```diff
@@ -108,10 +71,6 @@ public function getConfigTreeBuilder(): TreeBuilder
                         ->scalarNode('remove_token_from_body')->defaultTrue()->end()
                     ->end()
                 ->end()
-                ->scalarNode('logout_firewall')
```

Comment:
We should depreciate it before we delete it

Reply:
Yeah, thanks for the reminder here.  There are definitely a few things that could use a proper deprecation in a 1.x release that I'm now forgetting about since I've been sitting on this branch for so long 😬

Reply:
#350 for compat layer